custom_components/tuya_local/devices/sencor_air_conditioner.yaml

```diff
@@ -0,0 +1,85 @@
+name: Sencor Air Conditioner
+products:
+  - id: ypgwqvhjrl5chq6h
+    model: SAC MT7048C Air Conditioner
+primary_entity:
+  entity: climate
+  translation_key: aircon_extra
+  icon: "mdi:air-conditioner"
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - dps_val: false
+          value: "off"
+          icon: "mdi:hvac-off"
+        - dps_val: true
+          constraint: mode
+          conditions:
+            - dps_val: "0"
+              icon: "mdi:snowflake"
+              value: cool
+            - dps_val: "3"
+              icon: "mdi:fan"
+              value: fan_only
+            - dps_val: "2"
+              icon: "mdi:water"
+              value: dry
+    - id: 2
+      name: temperature
+      type: integer
+      range:
+        min: 17
+        max: 30
+      unit: C
+    - id: 3
+      name: current_temperature
+      type: integer
+    - id: 4
+      name: mode
+      type: string
+      hidden: true
+    - id: 5
+      name: fan_mode
+      type: string
+      mapping:
+        - dps_val: low
+          value: low
+        - dps_val: high
+          value: high
+          constraint: preset_mode
+          conditions:
+            - dps_val: true
+              invalid: true
+    - id: 19
```

Comment:
You should have either a temperature_unit dp, or a fixed unit attribute on the temperature, not both.  The temperature unit dp must return either "C" or "F" to Home Assistant (case sensitive), it the device is using lower case, this requires a mapping.